include/ur_client_library/control/script_command_interface.h

```diff
@@ -226,6 +232,19 @@ class ScriptCommandInterface : public ReverseInterface
     SET_MAX_JOINT_TORQUES = 9,      ///< Set max joint torques
   };
 
+  void warnIfNotSupported(const VersionInformation& min_polyscope5, const VersionInformation& min_polyscopeX,
```

Comment:
Is there a reason for implementing this function in header? In my opinion I would put it into the cpp file.